doc/releases/changelog-dev.md

```diff
@@ -460,6 +460,43 @@ of operators. [(#2622)](https://github.com/PennyLaneAI/pennylane/pull/2622)
   [(#2744)](https://github.com/PennyLaneAI/pennylane/pull/2744)
   [(#2767)](https://github.com/PennyLaneAI/pennylane/pull/2767)
 
+* The argument `argnum` of the function `qml.batch_input` has been redefined: now it indicates the
+  indices of the non-trainable batched parameters in the quantum tape. Consequently, its default
```

Comment:
Very minor: Could this be misunderstood by users to only count the non-trainable parameters, i.e. to use `argnum=0` with
```py
circuit(a_trainable_param, the_batched_non_trainable_param)
```
:thinking: 

Maybe:
```suggestion
  indices of the batched parameters, which need to be non-trainable, in the quantum tape. Consequently, its default
```